bin/strap.sh

```diff
@@ -284,6 +284,13 @@ else
   logk
 fi
 
+# Halt if requested before Daptiv-specific setup
```

Comment:
why would we want this? the whole point of strap is every machine gets setup the same way.

Reply:
@jtrinklein See notes 2 and 1.